main.py

```diff
@@ -0,0 +1,9 @@
+import Parser
+import commands_function
+
+
+if __name__=='__main__':
+    args=Parser.parseInput()
+    command =commands_function.commandfactory.build_command(args)
```

Comment:
you can use
"from commands_function import commandfactory"
to call commandfactory directly without referring to the file name

Reply:
same with Parser

Reply:
this is a good one, I didn't know about it before.
I will.